app/src/main/java/it/quip/android/actvitiy/CreateCircleActivity.java

```diff
@@ -0,0 +1,76 @@
+package it.quip.android.actvitiy;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import java.util.List;
+
+import it.quip.android.QuipitApplication;
+import it.quip.android.R;
+import it.quip.android.fragment.CreateCircleFragment;
+import it.quip.android.fragment.InviteFriendsFragment;
+import it.quip.android.model.Circle;
+import it.quip.android.model.User;
+import it.quip.android.util.MockUtils;
+
+public class CreateCircleActivity extends AppCompatActivity {
+
+    public static final String CREATED_CIRCLE = "it.quip.android.CREATED_CIRCLE";
+
+    CreateCircleFragment createCircleFragment;
+    InviteFriendsFragment inviteFriendsFragment;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_create_circle);
+
+        if (null == savedInstanceState) {
```

Comment:
So that you don't end up with this mistake 

```java
if (savedInstanceState = null)
```

The null in the front wont compile if you do a single = instead of ==